galleon-pack/galleon-shared/src/main/resources/modules/system/layers/base/io/opentelemetry/api/main/module.xml

```diff
@@ -9,16 +9,14 @@
 
     <resources>
         <artifact name="${io.opentelemetry:opentelemetry-api}"/>
-        <artifact name="${io.opentelemetry:opentelemetry-api-events}"/>
-        <artifact name="${io.opentelemetry:opentelemetry-semconv}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-api}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations}"/>
-        <artifact name="${io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations-support}"/>
     </resources>
 
     <dependencies>
+        <module name="io.opentelemetry.api.events" export="true"/>
```

Comment:
Why export? Why not just add the dep to the deployment via the DUP?

Reply:
Updated.